tools/travis/build.sh

```diff
@@ -12,6 +12,7 @@ tools/build/scanCode.py .
 cd $ROOTDIR/ansible
 
 ANSIBLE_CMD="ansible-playbook -i environments/local -e docker_image_prefix=testing"
+GRADLE_PROJS_SKIP="-x :core:pythonAction:distDocker  -x :core:python2Action:distDocker -x :core:swift3Action:distDocker"
```

Comment:
Java currently being tested
I can add here but would need to update the testLean task to excluded all Java tests also.